test/ExtendedXmlSerializerTest/TestObject/TestClassCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// MIT License
+// 
+// Copyright (c) 2016 Wojciech Nagórski
```

Comment:
Oh... no, that is going to take a little while longer.  I am all up to date on this PR, however.  I would recommend getting this one accepted and then we can look at the next one once it is complete. 👍